project/plugins.sbt

```diff
@@ -5,4 +5,5 @@ addSbtPlugin("com.localytics" % "sbt-dynamodb" % "2.0.3")
 addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat" % "0.1.14")
 addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
 addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.2.7")
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
\ No newline at end of file
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.7.6")
```

Comment:
[1.8.0](https://github.com/sbt/sbt-native-packager/releases/tag/v1.8.0) is available